src/fidesops/service/connectors/sql_connector.py

```diff
@@ -183,6 +183,32 @@ def create_client(self) -> Engine:
             echo=not self.hide_parameters,
         )
 
+    # Overrides BaseConnector.cursor_result_to_rows
+    @staticmethod
+    def cursor_result_to_rows(results: LegacyCursorResult) -> List[Row]:
+        """Convert SQLAlchemy results to a list of dictionaries"""
+        columns: List[Column] = results.cursor.description
+        l = len(columns)
+        rows = []
+        for row_tuple in results:
+            rows.append({columns[i][0]: row_tuple[i] for i in range(l)})
+        return rows
```

Comment:
I see this is elsewhere, but Python's `range` creates a sequence of numbers that we're iterating over, but we already have the columns themselves that we can use to iterate over, so it feels a bit roundabout.

 I might rewrite as:
```python
rows.append({col[0]: row_tuple[count] for count, col in enumerate(columns)})
```
where `enumerate` gives you both the column value whose loop we're on, as well as the current count.

Reply:
niiice, learning lots of new python things today!